src/js/tabs/led_strip.js

```diff
@@ -848,11 +865,11 @@ led_strip.initialize = function (callback, scrollPosition) {
 
 
         // set color modifiers (check-boxes) visibility
-        $('.overlays').hide();
-        $('.modifiers').hide();
-        $('.blinkers').hide();
-        $('.warningOverlay').hide();
-        $('.vtxOverlay').hide();
+        if (!areOverlaysActive(activeFunction)) $('.overlays').hide();
+        if (!areModifiersActive(activeFunction)) $('.modifiers').hide();
+        if (!areBlinkersActive(activeFunction)) $('.blinkers').hide();
+        if (!isWarningActive(activeFunction)) $('.warningOverlay').hide();
+        if (!isVtxActive(activeFunction)) $('.vtxOverlay').hide();
```

Comment:
```suggestion
        $('.overlays').toggle(!areOverlaysActive(activeFunction);
        $('.modifiers').toggle(!areModifiersActive(activeFunction);
        $('.blinkers').toggle(!areBlinkersActive(activeFunction);
        $('.warningOverlay').toggle(!isWarningActive(activeFunction);
        $('vtxOverlay').toggle(!isVtxActive(activeFunction);
```
Need to check !

Reply:
It's not working because this function gets called on every change on switches, I tested.

Reply:
My bad, need to remove the negation ( ! ) and also remove the lines https://github.com/betaflight/betaflight-configurator/pull/2996/files#diff-7fa9d993f9cf0be01c067a90e4edb501336ad7178b3a1a3106db0f6b7eb2e356L857-L871

Reply:
![image](https://user-images.githubusercontent.com/8344830/191935585-289bd0d0-8087-44b7-bd94-4f46add90123.png)

Reply:
Using toggle is a much better and dense solution,

Reply:
> 

I will check